gnocchi/rest/__init__.py

```diff
@@ -375,6 +375,21 @@ def MeasuresListSchema(measures):
         times.tolist(), values))
 
 
+VALID_TRANSFORMATION_METHODS = ["absolute", "negative"]
+
+
+def TransformSchema(transform):
```

Comment:
We parse a string and not a python structure. So pyparsing looks better for this use case.

Also the code is really simple as-is, adding pyparsing now make this change complex for no value.